scenarios/04_transitive_via_export/grandparent/BUILD

```diff
@@ -0,0 +1,47 @@
+package(default_visibility = ["//04_transitive_via_export:__subpackages__"])
+
+java_library(
+    name = "grandparent_lib",
+    srcs = glob(["src/main/java/**/*.java"]),
+    deps = ["@com_google_guava_guava//jar"],
+    exports = ["@com_google_guava_guava//jar"],
+)
+
+java_library(
+    name = "grandparent_tests_lib",
+    srcs = glob(["src/test/java/**/*.java"]),
+    deps=[
+      ":grandparent_lib",
+
+      "@org_junit_jupiter_junit_jupiter_api//jar",
+      "@org_junit_jupiter_junit_jupiter_engine//jar",
+      "@org_junit_platform_junit_platform_commons//jar",
+      "@org_junit_platform_junit_platform_console//jar",
+      "@org_junit_platform_junit_platform_engine//jar",
+      "@org_junit_platform_junit_platform_launcher//jar",
+      "@org_junit_platform_junit_platform_runner//jar",
+      "@org_opentest4j_opentest4j//jar",
+    ],
+)
+
+java_test(
+  name="grandparent_tests",
+  main_class="org.junit.platform.console.ConsoleLauncher",
+  args=[
+    "--select-package transitive_via_export.grandparent",
+    "--details verbose",
+  ],
+  runtime_deps=[
+    ":grandparent_tests_lib",
+    "@org_junit_jupiter_junit_jupiter_api//jar",
```

Comment:
this is already in :grandparent_tests_lib, is that intentional?

Reply:
`:grandparent_tests_lib` doesn't export that and I am using the api in the test code, so I have to declare this explicitly. new world :)